<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mdiWindow</class>
 <widget class="QMainWindow" name="mdiWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Empresa Tres Patitos</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>recursos/trespatitos.jpg</normaloff>recursos/trespatitos.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QMdiArea" name="mdiArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>541</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>500</width>
      <height>500</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>500</width>
      <height>500</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="mniMenubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>817</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="mniUsuarios">
    <property name="title">
     <string>Usuarios</string>
    </property>
    <addaction name="mniUsuario"/>
   </widget>
   <widget class="QMenu" name="mniEmpleados">
    <property name="title">
     <string>Empleados</string>
    </property>
    <addaction name="mniEmpleado"/>
   </widget>
   <widget class="QMenu" name="mniDepartamentos">
    <property name="title">
     <string>Departamentos</string>
    </property>
    <addaction name="mniDepartamento"/>
   </widget>
   <widget class="QMenu" name="mniBienes">
    <property name="title">
     <string>Bienes</string>
    </property>
    <addaction name="mniAsignacionBienes"/>
    <addaction name="mniDesligarBienes"/>
    <addaction name="mniInformacionDeBienes"/>
   </widget>
   <widget class="QMenu" name="mniReportes">
    <property name="title">
     <string>Reportes</string>
    </property>
    <addaction name="mniTabla_Reportes"/>
   </widget>
   <addaction name="mniUsuarios"/>
   <addaction name="mniEmpleados"/>
   <addaction name="mniDepartamentos"/>
   <addaction name="mniBienes"/>
   <addaction name="mniReportes"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="mniUsuario">
   <property name="text">
    <string>Usuario</string>
   </property>
  </action>
  <action name="mniEmpleado">
   <property name="text">
    <string>Empleados</string>
   </property>
  </action>
  <action name="mniDepartamento">
   <property name="text">
    <string>Departamentos</string>
   </property>
  </action>
  <action name="mniAsignacionBienes">
   <property name="text">
    <string>Asignacion de Bienes</string>
   </property>
  </action>
  <action name="mniTabla_Reportes">
   <property name="text">
    <string>Tabla Reportes</string>
   </property>
  </action>
  <action name="mniDesligarBienes">
   <property name="text">
    <string>Desligar Bienes</string>
   </property>
  </action>
  <action name="mniInformacionDeBienes">
   <property name="text">
    <string>Informacion de Bienes</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
